modules/auxiliary/scanner/http/fortios_vpnssl_traversal_leak.rb

```diff
@@ -0,0 +1,185 @@
+# frozen_string_literal: true
+
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Auxiliary::Report
+  include Msf::Auxiliary::Scanner
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Post::File
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name' => 'FortiOS Path Traversal Leak Credentials',
+      'Description' => %q{
+        FortiOS system file leak through SSL VPN via specially crafted HTTP
+        resource requests. A path traversal vulnerability in the FortiOS SSL
+        VPN web portal may allow an unauthenticated attacker to download FortiOS
+        system files through specially crafted HTTP resource requests.
+
+        This module reads logins and passwords in clear text from
+        the `/dev/cmdb/sslvpn_websession` file. This vulnerability affects
+        (FortiOS 5.4.6 to 5.4.12, FortiOS 5.6.3 to 5.6.7 and FortiOS 6.0.0
+        to 6.0.4).
+      },
+      'References' => [
+        ['CVE', '2018-13379'],
+        ['URL', 'https://www.fortiguard.com/psirt/FG-IR-18-384'],
+        ['EDB', '47287'],
+        ['EDB', '47288']
+      ],
+      'Author' => [
+        'lynx (Carlos Vieira)',         # initial module author from edb
+        'mekhalleh (RAMELLA Sébastien)' # this module author (Zeop Entreprise)
+      ],
+      'License' => MSF_LICENSE,
+      'DefaultOptions' => {
+        'RPORT' => 443,
```

Comment:
Change default option RPORT to 10443 the vendor default?

Reply:
yes, sure ! I forgeted that.